src/store/modules/farmer/types.ts

```diff
@@ -0,0 +1,41 @@
+import { CoreState, CoreData, PartialCoreData } from '../../types';
+
+/*----------------------- Models -----------------------*/
+
+interface BaseFarmer {
+  username: string;
+  firstName: string;
+  middleName: string;
+  lastName: string;
+  phoneCountry?: string;
+  phoneArea?: string;
+  phoneNumber?: string;
```

Comment:
I think `phoneNumber` should be required, no?

Reply:
Might well be, @enochtsang can you provide some feedback?